FluentHub/Views/UserPages/ProfilePage.xaml.cs

```diff
@@ -133,7 +134,7 @@ private async Task<bool> SetTopLevelUserInfo()
             }
 
             // Location
-            if (user.Location != "")
+            if (user.Location != null)
```

Comment:
No. it'll cause runtime error.
If `Blog` or `Location` is not set, the default value is "" instead of null. But the `Company` was null.
It's very annoying :((